adapters/zwave/index.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+var loadZWaveAdapters = require('./zwave-adapter.js');
```

Comment:
What's the purpose of this file? Doesn't AdapterManager just load ZWaveAdapter directly?